code/modules/clothing/under/ties.dm

```diff
@@ -312,7 +355,6 @@
 	desc = "Damn, it feels good to be a gangster."

```

Comment:
Yeah, these things were originally like mini ablatives

Once this goes through I could probably make a gang variant with minor armor or whatever